Classes/PackageFactory/AtomicFusion/Forms/Domain/Service/Runtime/FormRuntime.php

```diff
@@ -44,25 +52,25 @@ class FormRuntime implements FormRuntimeInterface
 
     /**
      * @Flow\Inject
-     * @var Task\ProcessTaskInterface
+     * @var Task\ProcessTask
```

Comment:
I'm not sure here - it might be better to inject the `TaskInterface` and configure the concrete Classes via `Objects.yaml`